lib/actions/local/configure.coffee

```diff
@@ -95,100 +67,65 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+joinPath = (path) -> '/' + path.join('/')
+
+load = (target, type) ->
+	{ Reconfix } = require('reconfix-preview')
```

Comment:
Is there any reason why the `reconfix-preview` package exports a `Reconfix` property with the actual class? Can we export the class directly?

Reply:
Yes, we can. When I first wrote the export, I thought the module had to be a map.